### PR TITLE
update namespace

### PR DESCRIPTION
**Pull request by Alice1319**
fixes：
https://github.com/eclipse-xpanse/xpanse/issues/2188

### CODE REVIEW

**Comment by WangLiNaruto** [comment]
LGTM

**Comment by wuzhuanhong** [comment]
LGTM

**Comment by yuanxiaojun987** [comment]
LGTM

File: zitadel/README.md [hunk 1]
```diff
@@ -214,12 +214,12 @@ Grant role `isv` of the project `eclipse-xpanse` to the new isv user.
 
 #### Edit Metadata Of New Isv User
 
-Edit the metadata of the new isv user. Add key `namespace` with fixed value into the metadata.
-![edit_isv_user_metadata_add_namespace](static/edit_isv_user_metadata_add_namespace.png)
+Edit the metadata of the new isv user. Add key `serviceVendor` with fixed value into the metadata.
+![edit_isv_user_metadata_add_serviceVendor](static/edit_isv_user_metadata_add_serviceVendor.png)
```
Comment (baixinsui) [inline zitadel/README.md]: According to the requirements, the field set in IAM should be `isv` not `serviceVendor`
image

File: zitadel/static/edit_isv_user_metadata_add_serviceVendor.png
Comment (baixinsui) [inline zitadel/static/edit_isv_user_metadata_add_serviceVendor.png:1]: Please update `namespace` to `isv` in the METADATA of all users after the code of xpanse is merged.

Comment (Alice1319) [reply zitadel/static/edit_isv_user_metadata_add_serviceVendor.png:1]: yes, will do this after xpanse code merged.

File: zitadel/README.md (hunk 1 above)
Comment (Alice1319) [reply zitadel/README.md]: fixed.

**Review by swaroopar — APPROVED** [review]

**Pull request merged**